src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,70 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+  public static final String DEFAULT_DELIMITER = ",|:";
+  public static final String REGEX = "//(.)\n(.*)";
```

Comment:
```suggestion
  private static final String DEFAULT_DELIMITER = ",|:";
  private static final String REGEX = "//(.)\n(.*)";
```
상수 활용 👍 

외부에서 사용되지 않는 상수는 private으로 막아주면 좋을 것 같아요 😄